docs/glossary.md

```diff
@@ -14,27 +14,27 @@ To help you navigate the platform more effectively, familiarize yourself with th
  | **KubeRocketCI Business Entity** - a part of the CI/CD process (the integration, delivery, and deployment of any codebase changes) |**Application** - a codebase type that is built as the binary artifact and deployable unit with the code that is stored in VCS. As a result, the application becomes a container and can be deployed in an environment.|
  |                                                                                                                           |**Autotests** - a codebase type that inspects a product (e.g. an application set) on a Deployment Stage. Autotests are not deployed to any container and launched from the respective code repository.|
- |                                                                                                                           |**CD Pipeline (Continuous Delivery Pipeline)** - business entity that describes the whole delivery process of the selected application set via the respective stages. The main idea of the CD pipeline is to promote the application version between the stages by applying the sequential verification (i.e. the second stage will be available if the verification on the first stage is successfully completed). _**NOTE**: The CD pipeline can include the essential set of applications with its specific stages as well._|
- |                                                                                                                           |**CD Pipeline Stage** - business entity that is presented as the logical gate required for the application set inspection. Every stage has one OpenShift project where the selected application set is deployed. All stages are sequential and promote applications one-by-one.|
+ |                                                                                                                           |**Deployment Flows** - business entity that describes the whole delivery process of the selected application set via the respective environments. The main idea of the CD pipeline is to promote the application version between the environments by applying the sequential verification (i.e. the second environment will be available if the verification on the first environment is successfully completed). _**NOTE**: The Deployment Flow can include the essential set of applications with its specific environments as well._|
+ |                                                                                                                           |**Deployment Flow Environment** - business entity that is presented as the logical gate required for the application set inspection. Every environment has one OpenShift project where the selected application set is deployed. All the environments are sequential and promote applications one-by-one.|
```

Comment:
Environment - 
please also remove OpenShift

Reply:
Environment - a logical entity that facilitates the inspection of application sets. Each environment is linked to a distinct Kubernetes namespace dedicated to containerized workloads, ensuring a one-to-one mapping between the environment and the namespace. This structure allows for the sequential promotion of applications from one environment to the next.